src/main/resources/sql/data.sql

```diff
@@ -0,0 +1,17 @@
+insert into RACE_RESULT
+values (1, 10, current_timestamp);
+insert into CAR
+values (1, '우르', 9, 1, 1, current_timestamp);
+insert into CAR
+values (2, '빙봉', 10, 1, 0, current_timestamp);
```

Comment:
직접 데이터를 넣으신거라 잘못 넣으셨을거라고 생각하는데요. bool type은 true가 1, false가 0입니다. 아래처럼 수정이 필요합니다.
```suggestion
insert into CAR
values (1, '우르', 9, 1, 0, current_timestamp);
insert into CAR
values (2, '빙봉', 10, 1, 1, current_timestamp);
```

기왕 바꾸는 거 우르가 우승하도록 하시죠ㅎㅎ

```suggestion
insert into CAR
values (1, '우르', 10, 1, 1, current_timestamp);
insert into CAR
values (2, '빙봉', 9, 1, 0, current_timestamp);
```

Reply:
test에 있는 데이터는 수정됐는데 main 은 안되있네요,, 빙봉이 이기는 것으로 수정하겠습니다 !! ㅋㅋㅋ

이렇게 헷갈리는거면 아래의 빙봉 코멘트처럼 `1, 0` 보다는 `y, n`으로 구분해주는게 더 가독성이 좋아보입니다.